packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -211,8 +238,14 @@ class RenderAndroidView extends RenderBox {
 }
 
 class _AndroidViewGestureRecognizer extends OneSequenceGestureRecognizer {
-  _AndroidViewGestureRecognizer(this.dispatcher, List<OneSequenceGestureRecognizer> gestureRecognizers) {
-    this.gestureRecognizers = gestureRecognizers;
+  _AndroidViewGestureRecognizer(this.dispatcher, this.gestureRecognizerFactories) {
+    _gestureRecognizers = gestureRecognizerFactories
+        .map((Factory<OneSequenceGestureRecognizer> factory) => factory.constructor()).toSet();
+    team = GestureArenaTeam();
+    team.captain = this;
+    for (OneSequenceGestureRecognizer recognizer in _gestureRecognizers) {
+      recognizer.team = team;
+    }
```

Comment:
you could reduce this to one loop:

```dart
team = GestureArenaTeam();
￼team.captain = this;
_g￼estureRecognizers = gestureRecognizerFactories.map(
  (Factory<OneSequenceGestureRecognizer> factory) {
    return factory.constructor()
      ..team = team;
  }
).toSet();
```

or:

```dart
team = GestureArenaTeam();
￼team.captain = this;
_gestureRecognizers ??= new Set<OneSequenceGestureRecognizer>();
_gestureRecognizers.clear();
for (Factory<OneSequenceGestureRecognizer> factory in gestureRecognizerFactories) {
  _gestureRecognizers.add(
    factory.constructor()
      ..team = team
  );
}
```

...or some such.

Reply:
done